src/test/resources/hudson/plugins/ec2/Ami.yml

```diff
@@ -4,7 +4,7 @@ configuration-as-code:
 jenkins:
   clouds:
     - amazonEC2:
-        cloudName: "test"
+        name: "test"
```

Comment:
Similar issue also exists in the integration tests for `configuration-as-code-plugin`